demo/src/main/java/com/novoda/demo/MainActivity.java

```diff
@@ -46,6 +33,7 @@ protected void onCreate(Bundle savedInstanceState) {
         View videoSelectionButton = findViewById(R.id.button_video_selection);
         View audioSelectionButton = findViewById(R.id.button_audio_selection);
         View subtitleSelectionButton = findViewById(R.id.button_subtitle_selection);
+        final ControllerView controllerView = (ControllerView) findViewById(R.id.controller_view);
```

Comment:
Doesn't look like it needs to be `final`

Reply:
1263d1f30dc497d1fbf5909674f9677bc7ddbb9a